### PR TITLE
Use nvm to install Node.js

### PR DESCRIPTION
**Pull request by agarzola**
## Description

This PR replaces our current strategy, where we install Node.js v10 directly, with an nvm-based solution, where we first install nvm and then use nvm to install v10. We also install the latest LTS version of Node.

## Motivation / Context

The intent here is to make nvm available to all consumers of our `php-apache` images so that individual projects can track their required version of Node.js in an `.nvmrc` file and use the nvm command to install said version.

In addition to that goal, this PR also preemptively installs Node v10 and the LTS version:
- v10 is installed and made the default version of Node in an effort to stay backwards compatible with projects that currently rely on v10 already being installed on this image. The idea is to eventually move all of our projects to either use LTS by default or declare and install their own version of Node via nvm, so the lines that install, use, and set v10 as the default will be removed eventually.
- LTS is installed so that projects using the latest version of Node get the benefit of already having LTS installed and thus running `nvm install && nvm use` in those projects will exit relatively quickly. This should help shorten build times in environments that are built from scratch (production, Tugboat base previews, Lando containers, etc).

## Testing Instructions / How This Has Been Tested

@markdorison I’ll need your assistance testing this to ensure it doesn’t break things on projects relying on Node.js v10 and that nvm is indeed installed and working correctly on projects moving to that strategy.

## Screenshots

n/a

## Documentation

- Should we document our Node.js installation strategy in the README?

### CODE REVIEW

**Comment by agarzola** [comment]
@markdorison `php-apache` images are not building due to nvm not finding `lts`. I searched online and found nvm-sh/nvm#1591, which indicates that either SSL certificates aren’t up to date or the system clock is off:
>it's likely that your system's SSL certificate authorities aren't up to date (or your system clock is wrong).

Incidentally: I’m curious why we run a single long `RUN` command instead of multiple discrete `RUN` commands in our Dockerfiles.

**Comment by markdorison** [comment]
@agarzola I pushed a fix for the `lts` issue.

I don't think this will break anything since `node` and `npm` should still be available as they were before, right?

**Comment by agarzola** [comment]
> I don't think this will break anything since `node` and `npm` should still be available as they were before, right?

They _should_, but I’d love to validate that assertion if possible.

**Review by markdorison — APPROVED** [review]

**Comment by agarzola** [comment]
As discussed earlier today, I will wait until next Monday (September 27) to merge this PR to avoid breaking Tugboat builds across the land at the end of the week. Once this is merged and a new image produced, I will take the following steps:
1. Create a new Tugboat preview (without a base preview) for chromatichq/chromatichq.com#3169 to validate that it works as expected (i.e. that Tugboat can simply invoke `nvm install` so that nvm can install v14 of Node).
2. Create a new Tugboat preview (without a base preview) for a test branch in `benz-platform` to validate that Node v10 is in fact available to it as is expected.
3. If either validation step above does not work, roll back the changes in this PR to ensure that nightly Tugboat base preview rebuilds do not break.

**Comment by agarzola** [comment]
I have updated this PR with `main` and added a commit to remove `nodejs` from the list of packages installed via `apt-get` (since it is now managed via nvm). I do not see a way to run the Build Docker Images action on command, so I’m going to skip step 1, merge this PR as soon as checks pass, and if anything goes wrong, submit a follow-up PR reverting the changes in this PR.

**Pull request merged**
